frontend/src/assets/index.css

```diff
@@ -3,13 +3,6 @@
 .ant-space-item {
 	margin-right: 0 !important;
 }
-.instrument-card{
```

Comment:
Why are we removing this?

Reply:
This is basically related to the #202 PR, which I raised few days back and it got merged. I actually forgot to remove this piece of code from that PR.